app/main.py

```diff
@@ -1,5 +1,64 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot():
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
```

Comment:
It's better to specify more detailed type annotation
```suggestion
    def __init__(self, name: str, weight: int, coords: Optional[list[int]] = None) -> None:
```

Also, please, note, if default value for the parameter is None - then this parameter should be marked as Optional. Please, fix it everywhere.

Reply:
Thanks! Прикольно, сейчас почитаю про Optional...